api-spec/STAC-extensions.yaml

```diff
@@ -761,35 +769,44 @@ components:
       properties:
         bbox:
           $ref: '#/components/schemas/bbox'
-    timeFilter:
-      description: An object representing a time based filter.
-      type: object
-      properties:
-        time:
-          $ref: '#/components/schemas/time'
     intersectsFilter:
       type: object
       description: Only returns items that intersect with the provided polygon.
       properties:
         intersects:
           $ref: 'https://geojson.org/schema/Geometry.json'
-    time:
+    datetimeFilter:
+      description: An object representing a datetime-based filter.
+      type: object
+      properties:
+        datetime:
+          $ref: '#/components/schemas/datetimeOrDatetimeInterval'
+    datetimeOrDatetimeInterval:
```

Comment:
we had two uses of this type, but one should only have allowed an RFC 3339 datetime, while the other allowed that or the `datetime/datetime` interval syntax, so split them into two definitions.

Reply:
This is how WFS defines it. https://raw.githubusercontent.com/opengeospatial/WFS_FES/master/core/openapi/parameters/datetime.yaml

Should we follow them?

Reply:
> Should we follow them?

I'll update this wrt their definition